src/components/common/index.ts

```diff
@@ -43,3 +43,4 @@ export * from './helpers/compareVersion'
 export * from './hooks/FileReader'
 export { default as FloatingVariablesSuggestions } from './FloatingVariablesSuggestions/FloatingVariablesSuggestions'
 export { default as HiddenInput } from './HiddenInput/HiddenInput'
+export * from './TLSConnectionForm'
```

Comment:
let's remove this as discussed

Reply:
If going with this approach should delete whole file and have it as a refactoring task.